controller/space.go

```diff
@@ -325,6 +330,9 @@ func ConvertSpace(request *goa.RequestData, p *space.Space, additional ...SpaceC
 		},
 		Links: &app.GenericLinks{
```

Comment:
Extend the GenericLinks to include the relatedWorkItemTypeList, same level as Self and remove it from Relations